storage/persistent_storage/persistent-storage-iscsi.adoc

```diff
@@ -31,8 +31,8 @@ By default, they are ports `860` and `3260`.
 
 [IMPORTANT]
 ====
-OpenShift assumes that all nodes in the cluster have already configured iSCSI
-initator, i.e. have installed `iscsi-initiator-utils` package and configured
+OpenShift assumes that all nodes in the cluster have already configured the iSCSI
+initiator, that is, they have installed the `iscsi-initiator-utils` package and configured
```

Comment:
Who is they? Can we swap they with "users" ? As it currently reads, it seems like the "nodes" have some autonomy.

Reply:
@skopacz1 When you [update the wording of the entire admonition](https://github.com/openshift/openshift-docs/pull/51070#issuecomment-1268423307), I think you need to s/OpenShift/{product-title}. I think we can merge this PR in the meantime.